src/utils/inputsExtractor.ts

```diff
@@ -49,7 +49,7 @@ function inputsParser(): IInputs {
     const pollingInterval: number = validateIntervalValues(
         parseFloat(core.getInput("polling_interval"))
     );
-    const retries: number = validateIntervalValues(parseInt(core.getInput("retries")));
+    const retries: number = parseInt(core.getInput("retries"));
 
```

Comment:
I guess it makes sense to let users choose the values they want but I still want to validate that the user doesn't accidentally provide us with a negative integer value that might cause issues later on, could we instead edit the `validateIntervalValues` method to remove the maximum value and simply return the provided value if it a positive number. Some checks will need to be updated with that change.

Reply:
Sounds good. I've adjusted it so `validateIntervalValues` is still used here, and instead `validateIntervalValues` no longer enforces the maximum value.

Reply:
I just pushed up one more tweak (I forgot to update the tests around this), but I guess one question about behavior is for `Infinity`. I changed it so `Infinity` returns `1` instead of the old max value, since it seems like `Infinity` is probably a mistake, but let me know if you'd prefer to do something else for the infinite case.

Reply:
Am good with that.